src/app/home/home.component.ts

```diff
@@ -104,7 +104,11 @@ export class HomeComponent implements AfterViewInit {
         }
     }
 
-    onKeyboardEvent($event: KeyboardEvent) {
+    /**
+     * Function that is called on a KeyboardEvent in the editor.
+     * @param $event {KeyboardEvent}
```

Comment:
```suggestion
     * @param {KeyboardEvent} $event 
```
This usually goes before as per [spec](https://google.github.io/styleguide/jsguide.html#jsdoc)